.github/workflows/fenicsx-tests.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         with:
           path: ./dolfinx
           repository: FEniCS/dolfinx
-          ref: main
+          ref: jpdean/manual_integration_domains_2
```

Comment:
don't forget to revert this before merge

Reply:
Is this needed to pass the integration test?

Reply:
Yes, but only because dolfinx main doesn't expect a list to be returned. I'll make a new branch of dolfinx now with a small change to account for this.